src/rpc/client.cpp

```diff
@@ -165,6 +165,7 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "logging", 0, "include" },
     { "logging", 1, "exclude" },
     { "spork", 1, "value" },
+    { "sporkupdate", 1, "value" },
```

Comment:
b94edd75b5a40f22252f91b8492bd3546cb6e521 needs a release notes, and really probably should be in it's own PR